sdk/communication/azure-communication-identity/src/test/java/com/azure/communication/identity/CommunicationIdentityAsyncTests.java

```diff
@@ -81,11 +77,45 @@ public void createUserWithResponse() {
     }
 
     @Test
-    public void createUserAndToken() {
+    public void createUserAndTokenWithChatScope() {
```

Comment:
80->128 is checking the same scenario but with different params, maybe we can eliminate duplication by using a parameterized test, wdyt?

Reply:
Updated the tests - used parameterized tests.
Also, instead of separate assertions in each tests to check user and token objects, used verifyTokenNotEmpty and verifyUserNotEmpty (created this one) methods.